src/BenchmarkDotNet/Portability/RuntimeInformation.cs

```diff
@@ -160,18 +160,14 @@ internal static CpuInfo GetCpuInfo()
             return null;
         }
 
-        internal static string GetNetCoreVersion() => InDocker ? GetDockerNetCoreVersion() : GetBaseNetCoreVersion();
+        internal static Version GetNetCoreVersion()
```

Comment:
Since this method does not return .NET Core version anymore I think that we should rename it to just `GetTargetFrameworkVersion`

```suggestion
        internal static Version GetTargetFrameworkVersion()
```